Solutions/Cloud Service Threat Protection Essentials/Hunting Queries/AzureResourceAssignedPublicIP.yaml

```diff
@@ -40,7 +40,7 @@ query: |
   | extend SchemaVersion = FASchemaVersion_s
```

Comment:
Can we consider using too many parse json to single statement .

  |extend d = tostring(parse_json(tostring(parse_json(Properties).responseBody)).properties)
  |extend c = parse_json(d) 
  | extend publicIPAddress_ = tostring(c.ipAddress) 
  | extend publicIPAddressVersion_ = tostring(c.publicIPAddressVersion) 
  | extend publicIPAllocationMethod_ = tostring(c.publicIPAllocationMethod)